app/cdap/components/WrangleHome/Components/WidgetSVG/utils.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import reject from 'lodash/reject';
+import { findHighestVersion } from 'services/VersionRange/VersionUtilities';
+
+export const getCategoriesToConnectorsMap = (connectionTypes = []) => {
+  const categoryToConnectionsMap = new Map();
+  const connectionToVersionsMap = new Map();
+  if (!connectionTypes.length) {
+    return categoryToConnectionsMap;
+  }
+  for (const connectionType of connectionTypes) {
```

Comment:
add some comments in here explaining whats going on

Reply:
we fetched versioning information from the existing code base & removed this utility function as it is redundant.